src/message/fetchActions.js

```diff
@@ -465,24 +461,23 @@ export const doInitialFetch = (): ThunkAction<Promise<void>> => async (dispatch,
       // use retrying; just log out.
       dispatch(logout());
     } else if (e instanceof Server5xxError) {
-      dispatch(initialFetchAbort('server'));
+      dispatch(registerAbort('server'));
     } else if (e instanceof NetworkError) {
-      dispatch(initialFetchAbort('network'));
+      dispatch(registerAbort('network'));
     } else if (e instanceof TimeoutError) {
       // We always want to abort if we've kept the user waiting an
       // unreasonably long time.
-      dispatch(initialFetchAbort('timeout'));
+      dispatch(registerAbort('timeout'));
     } else {
-      dispatch(initialFetchAbort('unexpected'));
+      dispatch(registerAbort('unexpected'));
       logging.warn(e, {
         message: 'Unexpected error during /register.',
       });
     }
     return;
   }
+  dispatch(registerComplete(initData));
```

Comment:
A good small other commit to add would be to move this action creator into this file, next to its "start" and "abort" counterparts. It's currently in `realmActions.js` (the only thing there). Which sort of makes sense if you accept the old "realm init" name, but doesn't make sense in any other way.